smart_contract/derive/src/entrypoint.rs

```diff
@@ -29,7 +29,9 @@ pub fn impl_entrypoint(_attr: TokenStream, item: TokenStream) -> TokenStream {
     block.stmts.insert(
         0,
         parse_quote!(
-            use ::iroha_wasm::{debug::DebugExpectExt as _, ExecuteOnHost as _, QueryHost as _};
+            use ::iroha_wasm::{
```

Comment:
```suggestion
            use ::iroha_smart_contract::{
```

shouldn't it be this?

Reply:
weird, does none of our tests use this macro?

Reply:
> weird, does none of our tests use this macro?

Looks like so

Reply:
Created #3988